Make repository installable with Vim Package Managers

gitmalet opened the pull request:
I moved the plugin files into a structure that is recognized by many package manager for `vim`.
The change is tested with [plug](https://github.com/junegunn/vim-plug). The plugin installs and at least syntax highlighting is active afterwards.

Please consider merging to make this plugin easier to install.

lifepillar commented:
This repo is structured as a Vim *package*, which is a collection of Vim plugins (see `:help packages`). So, you should be able to install each plugin by pointing your favorite package manager to the corresponding URL: have you tried that?

With vim-plug you should be able to install the repo to a custom location: so, something like this may work:

    Plug 'lifepillar/vim-formal-package' { dir: '~/.vim/pack/plugins/start' }

Or, you may use a package manager that has explicit support for Vim packages, such as [minpac](https://github.com/k-takata/minpac). There are possibly others (I don't use package managers).

I am not against merging this, but I'd prefer to keep the current structure, as it makes the most sense to me.

gitmalet commented:
Oh, was not aware of the concept of a vim package, that makes sense then.

I changed the approach and added a script that converts the package to something like a single plugin so that it can be installed with 'plug' for example.
The `Readme.md` describes the installation method.
It is a hacky solution, but it works and does not change the structure of the package (at least not in the repository).

lifepillar commented:
I have merged this: thanks for your contribution!
